netbout-web/src/main/java/com/netbout/rest/Markdown.java

```diff
@@ -96,7 +97,7 @@ public String html() {
         synchronized (Markdown.TIDY) {
             return LINK_WHITESPACE.matcher(
                 Markdown.clean(
-                    new PegDownProcessor().markdownToHtml(
+                    new PegDownProcessor(Extensions.ALL).markdownToHtml(
```

Comment:
@yegor256 Is there any reason why you didn't use the Pegdown extensions before?
I mean if we will use `Extensions.ALL` here, it will have no impact on the Netbout agents?